docs/examples/009-subsetting.ipynb

```diff
@@ -0,0 +1,397 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    import geocat.datafiles as geodf
</pre><p> </p><p>Maybe <code>gdf</code> instead of <code>geodf?</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/616/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
`gdf` would be confused with GeoDataFrame as per our other examples

Reply:
Oh I see; I hadn't noticed that. Okay, even leaving it as is should be just fine with me!